hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -110,8 +115,10 @@ import qualified Control.Monad.Trans.Writer.Strict as Strict
 
 import qualified Data.Char as Char
 import           Data.Functor.Identity (Identity(..))
+import           Data.Map (Map)
+import qualified Data.Map as Map
```

Comment:
You may want to use the strict map functions here I think.
```haskell
import           Data.Map (Map)
import qualified Data.Map.Strict as Map
```

Reply:
Sorry I think @felixmulder misunderstood my comment.

I suggested importing just Data.Map for the `Map` type.

What @jystic has written is what I was thinking.